Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
Even if you make it public, I don't think MenuView's layout works properly.
I'm working on this with https://github.com/kitasuke/PagingMenuController/pull/197.
Is this change relevant to multiline text menu?